### DISCUSSION TITLE
locales not working with offline mode

### DISCUSSION THREAD

**Discussion by martin960** [requarks/wiki#3381] [Help / Questions]
**Question**
Hi,im trying to install wiki.js with offline mode enabled. 
Everything works except i dont have any locales working, so for example text field look like:
"general.subtitle"
So then i tried sideloading the locales and put locales.json and en.json in the sideload directory.
Now i see the following error in the log:
```
info: Sideload directory detected. Looking for packages...
info: Found locales master file. Importing locale packages...
warn: /wiki/data/sideload/locales.json: Unexpected token < in JSON at position 5
```
I used the newest locales.json out of the master branch.

Am i doing something wrong?

Any help is appreciated

Kind regards
Martin  

**Host Info (please complete the following information):**
OS: Docker
Wiki.js version: 2.3.81
Database engine: postgres

**Comment by BillPelegrini** [comment]
Same problem here.

`Jun  3 15:50:13 wikidbapp nodejs-wikidb: 2020-06-03T18:50:13.201Z [MASTER] ^[[32minfo^[[39m: Sideload directory detected
. Looking for packages...
Jun  3 15:50:13 wikidbapp nodejs-wikidb: 2020-06-03T18:50:13.204Z [MASTER] ^[[32minfo^[[39m: Found locales master file. Importing locale packages...
Jun  3 15:50:13 wikidbapp nodejs-wikidb: 2020-06-03T18:50:13.228Z [MASTER] ^[[33mwarn^[[39m: /data/wiki/data/sideload/locales.json: Unexpected token < in JSON at position 5`

**Comment by NGPixel** [comment]
Make sure your locales.json is identical to https://raw.githubusercontent.com/Requarks/wiki-localization/master/locales.json
